### PR TITLE
Return indices for further logging.

### PR DESCRIPTION
**Pull request by vcarpani**
This PR adds mask indices returned value to draw mask. It could be used for further logging.

### CODE REVIEW

File: keras_maskrcnn/utils/visualization.py [hunk 1]
```diff
@@ -42,8 +42,11 @@ def draw_mask(image, box, mask, label=None, color=None, binarize_threshold=0.5):
     image[indices[0], indices[1], :] = 0.5 * image[indices[0], indices[1], :] + 0.5 * mask[indices[0], indices[1], :]
 
     # draw the border
-    indices = np.where(border != [0, 0, 0])
-    image[indices[0], indices[1], :] = 0.2 * image[indices[0], indices[1], :] + 0.8 * border[indices[0], indices[1], :]
+    border_indices = np.where(border != [0, 0, 0])
+    image[border_indices[0], border_indices[1], :] = 0.2 * image[border_indices[0], border_indices[1], :] + 0.8 * border[border_indices[0], border_indices[1], :]
+
+    return indices
+
```
Comment (vsuryamurthy) [inline keras_maskrcnn/utils/visualization.py]: Remove extra line

File: keras_maskrcnn/utils/visualization.py [hunk 2]
```diff
@@ -42,8 +42,11 @@ def draw_mask(image, box, mask, label=None, color=None, binarize_threshold=0.5):
     image[indices[0], indices[1], :] = 0.5 * image[indices[0], indices[1], :] + 0.5 * mask[indices[0], indices[1], :]
 
     # draw the border
-    indices = np.where(border != [0, 0, 0])
-    image[indices[0], indices[1], :] = 0.2 * image[indices[0], indices[1], :] + 0.8 * border[indices[0], indices[1], :]
+    border_indices = np.where(border != [0, 0, 0])
+    image[border_indices[0], border_indices[1], :] = 0.2 * image[border_indices[0], border_indices[1], :] + 0.8 * border[border_indices[0], border_indices[1], :]
+
+    return indices
```
Comment (vsuryamurthy) [inline keras_maskrcnn/utils/visualization.py]: Update the documentation above with Returns in draw_mask and draw_masks.

**Review by vsuryamurthy — APPROVED** [review]

**Comment by vcarpani** [comment]
Solved ;)

**Comment by hgaiser** [comment]
Could you check the CI? Other than that LGTM.

**Pull request merged**